citations-tool/tool/src/java/org/sakaiproject/citation/tool/CitationHelperAction.java

```diff
@@ -1094,7 +1094,8 @@ protected void doPostJsonResponse(ParameterParser params,
 						.replaceAll("\\{\"children\":\\[\\[", "{\"children\":[")
 						.replaceAll("\\}\\],\\[\\{\"section", "},{\"section")
 						.replaceAll("\\}\\]\\]", "}]")
-						.replaceAll("\\}\\],\\[\\]\\]", "}]");
+						.replaceAll("\\}\\],\\[\\]\\]", "}]")
```

Comment:
Yes and hard to maintain too. I need to put some thought into how to fix this..either by telling the plugin to ignore some of the unneeded ol's or fixing the data in the javascript.